src/integTest/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanelIntegTest.kt

```diff
@@ -98,7 +113,38 @@ class SnykToolWindowPanelIntegTest : HeavyPlatformTestCase() {
 
         assertFalse("Medium severity IaC results should NOT be shown after filtering", isMediumSeverityShown())
 
-        // restore modified Registry value
-        isIacEnabledRegistryValue.setValue(isIacEnabledOldValue)
+        tearDownIacTest()
+    }
+
+    @Test
+    fun testIacErrorShown() {
+        // pre-test setup
+        setUpIacTest()
+
+        // mock IaC results
+        val iacError = SnykError("fake error", "fake path")
+        val iacResultWithError = IacResult(null, iacError)
+
+        mockkStatic("io.snyk.plugin.UtilsKt")
+        every { getIacService(project).isCliInstalled() } returns true
+        every { getIacService(project).scan() } returns iacResultWithError
+
+        // actual test run
+        project.service<SnykTaskQueueService>().scan()
+
+        val toolWindowPanel = project.service<SnykToolWindowPanel>()
+        PlatformTestUtil.waitWhileBusy(toolWindowPanel.getTree())
+
+        assertEquals(iacError, toolWindowPanel.currentIacError)
+
+        TreeUtil.selectNode(toolWindowPanel.getTree(), toolWindowPanel.getRootIacIssuesTreeNode())
+        PlatformTestUtil.waitWhileBusy(toolWindowPanel.getTree())
+
+        val descriptionComponents = toolWindowPanel.getDescriptionPanel().components.toList()
+        assertTrue(descriptionComponents.any { it is SnykErrorPanel })
```

Comment:
Please check for the actual error content, to make sure it's not a different error displayed :). You can get the Label with UIComponentFinder class.